common/src/main/scala/explore/components/TileController.scala

```diff
@@ -37,22 +37,25 @@ final case class TileController(
   defaultLayout:    LayoutsMap,
   layoutMap:        View[LayoutsMap],
   tiles:            List[Tile],
+  section:          GridLayoutSection,
   clazz:            Option[Css] = None
 )(implicit val ctx: AppContextIO)
-    extends ReactProps[TileController](TileController.component)
+    extends ReactFnProps[TileController](TileController.component)
 
 object TileController {
   type Props = TileController
 
   implicit val propsReuse: Reusability[TileController] = Reusability.derive
 
-  def storeLayouts(userId: Option[User.Id], layouts: Layouts)(implicit
+  def storeLayouts(userId: Option[User.Id], section: GridLayoutSection, layouts: Layouts)(implicit
     ctx:                   AppContextIO
   ): Callback =
-    UserGridLayoutUpsert
-      .storeLayoutsPreference[IO](userId, GridLayoutSection.ObservationsLayout, layouts)
-      .runAsync
-      .debounce(1.second)
+    Callback.log(s"store $userId $section") *>
+      UserGridLayoutUpsert
+        .storeLayoutsPreference[IO](userId, section, layouts)
+        .runAsync
+        .handleErrorWith(r => Callback.log(r.getMessage))
```

Comment:
Since we have `AppContextIO` already in implicit context, we can use `Logger` here (and move `.runAsync` after error handling).